lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
Yeah, but I'm really not sure how we should do that. It feels somehow dirty that every sequelize instace should have its own instance of the promise prototype ...

On the other hand, is it really that important to support multiple namespaces? The namespace is only used by sequelize to store the transaction, so we can decide on whatever we want. If users are using another namespace to store their own values that is up to them :). I'm leaning towards always hardcoding the ns to `'sequelize'` actually.

If we change it to a flag `cls: true / false` we would still have the issue that the option applies globally, to all instances though